public/app/core/utils/explore.ts

```diff
@@ -192,7 +192,7 @@ export const safeParseJson = (text?: string): any | undefined => {
 };
 
 export const safeStringifyValue = (value: any, space?: number) => {
-  if (!value) {
+  if (value === undefined || value === null) {
```

Comment:
this slightly changes the behaviour of a bunch of things, like the panel model inspector in dashboards and variables, did we test it's safe? (especially for variables as `false` and `0` would have resulted in an empty string, but `"false"` and `"0"` now, not sure if that's even possible tho)

Reply:
I just couldn't imagine why we would want true to evaluate to "true" and false to evaluate to an empty string. Let's look at the usages of this function outside how it is used in this PR

- `features/dashboard/state/PanelModel.ts` - passes the entire model into this function, and that model is a required parameter. I am extremely confident that PanelModel will never evaluate to false or 0, there are a lot of required fields in it.
-  `features/variables/utils.ts` - safe-stringifies the first arg, concatenates all args together split by a space except the last one and checks that string has matches for any of the three formats for variables. I do not believe that having false or 0 evaluate to an empty string or not will have any impact on this, because the safe-stringify only runs on the first argument, and in all cases a variable reference must start with either `$` or `[` which is not possible with a false or 0 scenario.
- `features/variables/inspect/utils.ts` - Similar to the above, we safe-stringify a value and check if it matches any of the variable patterns. False or 0 or empty string will all not match. That value is not used again - it will use the matching groups instead.
- `plugins/datasource/prometheus/datasource.tsx` - This will only run if the if statement depending on the same value is true, so if the value passed in is 0 or false, it will never run in the first place